composer.json

```diff
@@ -19,8 +19,8 @@
     },
     "require": {
         "php": ">=5.4",
-        "robmorgan/phinx": "~0.5.0",
-        "cakephp/cakephp": "~3.1 || 3.2.*"
+        "robmorgan/phinx": "0.5.1",
+        "cakephp/cakephp": "~3.1"
```

Comment:
No, it means we support from 3.1.0 up to the current release (as long as the current release is < 4.0)